src/syngen/ml/vae/wrappers/wrappers.py

```diff
@@ -243,6 +243,14 @@ def fit_on_df(
 
         self.optimizer = self._create_optimizer()
         self.loss_metric = self._create_loss()
+
+        MlflowTracker().start_run(
```

Comment:
Is this "start_run" corresponds to "end_run" in dataset.py?

Reply:
No, this run corresponds to the "end_run" in "__infer_tables" of the class Worker